CHANGELOG.md

```diff
@@ -45,6 +45,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixed bugs
 
+* The built-in diff editor now correctly handles deleted files.
+
+* The built-in diff editor now correctly retains the executable bit on newly
+  added files when splitting.
```

Comment:
Can you include links to the fixed bugs?

Reply:
Done in https://github.com/jj-vcs/jj/pull/5486/commits/e1a5f583956fbaa2d771aa872fd1f5b11b27d088